### PR TITLE
chore(docs): Improve docstrings

### PR DESCRIPTION
**Pull request by Anavelyz**
## Description

* Fix DataFrame word in the docstrings
* Improving the docstrings

## Related Issue

#197 

## Checklist

- [x] I've read the [`CODE_OF_CONDUCT.md`](https://github.com/thegraphnetwork/epigraphhub_py/blob/master/CODE_OF_CONDUCT.md) document.
- [x] I've read the [`CONTRIBUTING.md`](https://github.com/thegraphnetwork/epigraphhub_py/blob/master/CONTRIBUTING.md) guide.
- [x] I've used pre-commit hook locally.
- [ ] I've written tests for all new methods and classes that I created.
- [ ] I've written the docstring in Google format for all the methods and classes that I used.
- [x] Semantic-release prefix title used (case sensitive):
  - Use the prefix `chore(docs):` for Examples / docs / tutorials / dependencies update
  - Use the prefix `fix:` for a bug fix
    (non-breaking change which fixes an issue).
  - Use the prefix `chore(improvement):` for an improvement
    (non-breaking change which improves an existing feature).
  - Use the prefix `feat:` for a new feature
    (non-breaking change which adds functionality).
  - Use the prefix `BREAKING CHANGE:` for a breaking change
    (fix or feature that would cause existing functionality to change).
  - Use the prefix `chore(security):` for a security fix.

### CODE REVIEW

File: epigraphhub/analysis/epistats.py [hunk 1]
```diff
@@ -70,20 +83,25 @@ def risk_ratio(
...
-        control_cases: number of cases in the control group
-        control_total: size of the control group
-    Returns:
-    RelativeRiskResult object
+    Also known as relative risk, it computed the risk of contracting a disease
+    given exposure to a risk factor.
+    
+    Parameters
+    ----------
+        exposed_cases: int
```
Comment (xmnlab) [inline epigraphhub/analysis/epistats.py:91]: do we need this indentation?

File: epigraphhub/data/sinan_fetch.py [hunk 2]
```diff
@@ -34,13 +34,18 @@ def download_data(disease: str, year: int) -> str:
...
     Download dataset and check if parquet directory exists.
 
-    -----
+    
     Parameters
-        disease: disease name.
-        year: available year.
-    ------
+    ----------
+        disease: str
```
Comment (xmnlab) [inline epigraphhub/data/sinan_fetch.py:40]: I think we have extra indentation here

**Review by xmnlab — COMMENTED** [review]
I haven't read it in details, but in general it looks good to me.
@eduardocorrearaujo could you review this one by chance? thanks

**Comment by xmnlab** [comment]
I think we probably should try to move this forward.

**Comment by Anavelyz** [comment]
What should I do to help with this?

**Comment by xmnlab** [comment]
@Anavelyz have you closed this on purpose?

**Comment by Anavelyz** [comment]
> @Anavelyz have you closed this on purpose?

No, I was trying to do a fetch and deleted the commits, then git didn't see any changes so it closed

**Review by xmnlab — CHANGES_REQUESTED** [review]
Thanks @Anavelyz for working on that. In general looks good. Just noted that some docstrings are not in the numpydocs standard. Could you move that docstrings to numpydics too please? Thanks!!

**Comment by Anavelyz** [comment]
> Thanks @Anavelyz for working on that. In general looks good. Just noted that some docstrings are not in the numpydocs standard. Could you move that docstrings to numpydics too please? Thanks!!

Ready!

File: epigraphhub/data/foph/viz.py [hunk 3]
```diff
@@ -79,39 +79,45 @@ def get_cluster_data(
...
+    vaccine : bool
+        If True the data of total vaccinations per hundred for the country in
+        the schema will be added in the final data frame. This data is from our
+        world in data.
+
+    smooth : bool
+        If True in the end data frame will be applied a moving average of seven
+        days.
+
+    :Returns :
```
Comment (xmnlab) [inline epigraphhub/data/foph/viz.py:119]: the returns sections looks wrong

**Review by xmnlab — CHANGES_REQUESTED** [review]
just one small comment inline. 
please fix that and ping me again and we can merge this pr. 
thanks!

**Review by xmnlab — APPROVED** [review]
LGTM! thanks for working on that

**Pull request merged**